buildSrc/src/main/java/org/elasticsearch/gradle/test/NamingConventionsCheck.java

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-package org.elasticsearch.test;
+package org.elasticsearch.gradle.test;
```

Comment:
Maybe this package move should be in its own PR.

Reply:
reverted